homeassistant/components/kmtronic/switch.py

```diff
@@ -52,6 +54,19 @@ def is_on(self):
             return not self._relay.is_energised
         return self._relay.is_energised
 
+    @property
+    def device_info(self):
```

Comment:
I think this should be using self._attr_device_info to avoid re computing it every time.

Reply:
opportunistically added other _attr_*